### DISCUSSION TITLE
[1817070] Marvel’s Spider-Man Remastered [漫威蜘蛛侠：重制版] [本体]

### DISCUSSION THREAD

**Discussion by makabakaye** [pjy612/SteamManifestCache#326] [Share]
[1817070] Marvel’s Spider-Man Remastered [漫威蜘蛛侠：重制版] [本体]
appid:1817070
Keys：
```
1817071;490a357029d1621deff9708e5d42f19324ccc758124ae1b455e091551df2a041
1817072;ae2589c0e9de7a18dea92ceac08a6537cf7b83c1983e9f4738a188a49fd137f9
1817074;d1d379b15dba708a435fa177bacfc7f625c53c84b45b299ded3cbeeca084e330
```
内测code:D40F286E5E323011
email:[211238482@qq.com]
[清单.zip](https://github.com/pjy612/SteamManifestCache/files/13858842/default.zip)

**Comment by pjy612** [comment]
请补上中文语言包的 密钥 应该是 1829896
https://steamdb.info/app/1817070/depots/?branch=public

**Reply by makabakaye** [reply]
游戏本体自带多国语言包括繁体中文但不包括简体中文，无需额外密钥，进游戏后就可以设定。而且steam游戏属性中也不支持语言的简体中文选项。

**Reply by pjy612** [reply]
好的 之前没注意 那个 depot 的容量

**Reply by pjy612** [reply]
查收
